packages/components/organisms/f-form/src/components/Form.vue

```diff
@@ -67,33 +105,112 @@ export default {
             });
 
             return formFields;
+        },
+
+        invalidFieldsSummary () {
+            const invalidFieldCount = this.$v.formFields.$dirty
+                && validations.getFormValidationState(this.$v.formFields).invalidFields.length;
+
+            if (!invalidFieldCount) return null;
+
+            return invalidFieldCount === 1
+                ? this.copy.errorMessages.singleFieldError
+                : this.copy.errorMessages.multipleFieldErrors.start + invalidFieldCount + this.copy.errorMessages.multipleFieldErrors.end;
```

Comment:
Instead of enforcing this kind of sentence structure could you try something like this, where you have a template and the number gets passed in?
https://kazupon.github.io/vue-i18n/guide/formatting.html#support-ruby-on-rails-i18n-format